lib/shelley/src/Cardano/Wallet/Shelley/MinimumUTxO.hs

```diff
@@ -67,46 +59,56 @@ computeMinimumCoinForUTxO = \case
 -- Importantly, a value that is valid in one era will not necessarily be valid
 -- in another era.
 --
-computeMinimumCoinForShelleyBasedEra
+computeMinimumCoinForUTxOShelleyBasedEra
     :: HasCallStack
     => MinimumUTxOForShelleyBasedEra
     -> Address
     -> TokenMap
     -> Coin
-computeMinimumCoinForShelleyBasedEra
+computeMinimumCoinForUTxOShelleyBasedEra
     (MinimumUTxOForShelleyBasedEra era pp) addr tokenMap =
-        extractResult $
+        unsafeCoinFromCardanoApiCalculateMinimumUTxOResult $
         Cardano.calculateMinimumUTxO era
             (embedTokenMapWithinPaddedTxOut era addr tokenMap)
             (Cardano.fromLedgerPParams era pp)
+
+-- | Returns 'True' if and only if the given 'TokenBundle' has a 'Coin' value
+--   that is below the minimum acceptable 'Coin' value.
+--
+isBelowMinimumCoinForUTxO
```

Comment:
Might be worth adding a comment here or for `computeMinimumCoinForUTxO` to highlight the difference between the two? E.g. adding this paragraph for the  `computeMinimumCoinForUTxO` docs:

> Not to be used for validation of outputs with user-provided 'Coin' values. For that, please use
> 'isBelowMinimumCoinForUTxO'.